udata/core/user/search.py

```diff
@@ -49,8 +49,8 @@ class UserSearch(ModelSearchAdapter):
         'views': Sort('metrics.views'),
         'created': Sort('created'),
     }
-    facets = {
-        'organization': ModelTermFacet('organizations', Organization),
+    aggregations = {
+        'user_organizations': ModelTermFacet('user_organizations', Organization),
```

Comment:
The name is used in URL so it will be `user_organization=orgid` same as dataset org, reuse org, tags...